procedures/katello/release.md.erb

```diff
@@ -22,16 +22,15 @@
 
 ## Release Packager
 
-* [ ] In foreman-packaging rpm/<%= short_version %> branch, change katello.spec, katello-repos.spec, rubygem-katello.spec:
+* [ ] Update `katello`, `katello-repos` and `rubygem-katello` in [foreman-packaging](https://github.com/theforeman/foreman-packaging) [rpm/<%= short_version %>](https://github.com/theforeman/theforeman-rel-eng/blob/master/bump_rpm_packaging) branch by either using the script or executing the steps manually:
```

Comment:
requires https://github.com/theforeman/theforeman-rel-eng/pull/5 and https://github.com/theforeman/theforeman-rel-eng/pull/6

Reply:
Technically, yes. I didn't have the time to investigate whether it actually works now ;)

Reply:
I'm leaning to only documenting the scripted steps and making sure it works during the next release.

Reply:
done